src/main/java/ua/lviv/iot/parkingServer/natscontroller/NatsParkingDeleteController.kt

```diff
@@ -18,9 +19,8 @@ class NatsParkingDeleteController(
 
     override val parser: Parser<DeleteParkingRequest> = DeleteParkingRequest.parser()
 
-    override fun generateReplyForNatsRequest(request: DeleteParkingRequest): DeleteParkingResponse {
-        val parkingId = request.parkingId
-        service.deleteEntity(parkingId)
-        return DeleteParkingResponse.newBuilder().build()
+    override fun generateReplyForNatsRequest(request: DeleteParkingRequest): Mono<DeleteParkingResponse> {
+        service.deleteEntity(request.parkingId).block()
+        return Mono.just(DeleteParkingResponse.getDefaultInstance())
```

Comment:
```suggestion
        return service.deleteEntity(request.parkingId)
            .thenReturn(DeleteParkingResponse.getDefaultInstance())
```
do not use block in your code, you're multiplying by zero whole async-non-blocking concepts. The only place where it is possible is the test code.

Reply:
✅